lib/drivers/tandemTslimDriver.js

```diff
@@ -1650,6 +1685,76 @@ module.exports = function (config) {
     return records;
   };
 
+  var buildAlarmRecords = function (data,records) {
```

Comment:
I think you forgot to call the `buildAlarmRecords` function down where the other builders are called. (Or at least on first test I got no `subType: 'alarm'` records uploaded, and I don't see it.